magefile.go

```diff
@@ -644,10 +649,18 @@ func packageAgent(requiredPackages []string, packagingFn func()) {
 		// cleanup after build
 		defer os.Unsetenv(agentDropPath)
 
-		packedBeats := []string{"filebeat", "heartbeat", "metricbeat", "osquerybeat"}
 		if devtools.ExternalBuild == true {
+			// for external go for all dependencies
+			dependencies := []string{
+				"auditbeat", "filebeat", "heartbeat", "metricbeat", "osquerybeat", "packetbeat", // beat dependencies
+				"apm-server",
+				// "cloudbeat", // TODO: add once working
```

Comment:
Cloudbeat v2 alignment was done [here](https://github.com/elastic/cloudbeat/pull/458). 
Anything else we need to do to remove this comment?

Reply:
I don't think there is anything blocking inclusion of Cloudbeat here, easiest fix is for you to submit a PR :)